listener/pkg/event/skr_events_listener.go

```diff
@@ -22,11 +21,6 @@ const (
 	requestSizeLimitInBytes = 16384 // 16KB
 )
 
-func RegisterListenerComponent(addr, componentName string, verify Verify) (*SKREventListener, *source.Channel) {
```

Comment:
This function is obsolete as it refers to a `source.Channel` type which is now a non-public type `source.channel`